bindings/java/src/main/com/apple/foundationdb/FDBTransaction.java

```diff
@@ -70,6 +70,16 @@ public void setReadVersion(long version) {
 			return getKey_internal(selector, true);
 		}
 
+		@Override
+		public CompletableFuture<Long> getEstimatedRangeSizeBytes(byte[] begin, byte[] end) {
```

Comment:
Not sure if a change is still coming, but just in case it was overlooked this note is still outstanding (and can now also be applied to the overload directly below as well).

Reply:
It was already changed to call the outer version. But I forgot to remove the lock acquiring and the try-catch part which is no longer necessary.